manifest.json

```diff
@@ -10,11 +10,13 @@
   },
   "mustUpdateAt": "2019-10-01",
   "billingOptions": {
-    "termsURL": "",
+    "type": "free",
     "support": {
       "url": "https://support.vtex.com/hc/requests"
     },
-    "free": true
```

Comment:
I think we should keep this, no?

Reply:
Following the [billingOptions docs](https://vtex.io/docs/concepts/billing-options/), I didn't find any info about this property, so that's why I removed it.

Reply:
Following the [billingOptions docs](https://vtex.io/docs/concepts/billing-options/), I didn't find any info about this property, so that's why I removed it.